helix-core/src/main/java/org/apache/helix/tools/ClusterVerifiers/BestPossibleExternalViewVerifier.java

```diff
@@ -84,7 +84,8 @@ public BestPossibleExternalViewVerifier(String zkAddr, String clusterName, Set<S
     _dataProvider = new ResourceControllerDataProvider();
   }
 
-  public BestPossibleExternalViewVerifier(HelixZkClient zkClient, String clusterName,
+  @Deprecated
```

Comment:
Nit: Maybe can we add javadoc for why it is deprecated and what is recommended to use?

Reply:
Added more javadoc.